.gitpod.yml

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# https://www.gitpod.io
+# https://www.gitpod.io/docs/41_Config_Gitpod_File/
+# https://github.com/gitpod-io/workspace-images/tree/master/mysql
+
+image: gitpod/workspace-mysql
+ports:
+- port: 3306
+  onOpen: ignore
+- port: 8005
+  onOpen: ignore
+- port: 8009
+  onOpen: ignore
+- port: 8080
+- port: 8081
+  onOpen: ignore
+- port: 8443
+tasks:
+- command: >
+      # similar to .travis.yml, but using authentication_string= instead of password= due to mysql major version difference
+      mysql -e "USE mysql;\nUPDATE user SET authentication_string=PASSWORD('mysql') WHERE user='root';\nFLUSH PRIVILEGES;\n";
+      ./gradlew createDB -PdbName=mifosplatform-tenants;
+      ./gradlew createDB -PdbName=mifostenant-default;
+      ./gradlew test
```

Comment:
@meysholdt do you happen to be able to spot what I'missing here, it's NOK and causes:

```
gitpod /workspace/fineract $ {
ifostenant-default; ./gradlew testlatform-tenants; ./gradlew createDB -PdbName=mi
> 
> }; HISTFILE=/workspace/.cmd-0 history -r
bash: syntax error near unexpected token `}'
gitpod /workspace/fineract $ 
```

Reply:
you may want to try `command: |` instead of `command: >`. The `|` causes the following string to be treated as multi-line. The `>` makes it single-line. 

With `command: |` the `;` at the end of the `bash` lines are no longer needed, e.g. in `./gradlew ... ;`

Reply:
I think this error is caused by the `#` comment inside the `command` (because Gitpod folds commands into one line, and doesn't currently support comments).

I tried this workaround: https://github.com/jankeromnes/fineract/commit/0830e34e4939df5d6c45bf46ea2a2b22355f24ba

which seems to make your `command` work in Gitpod.

Reply:
https://yaml-multiline.info :-D

Reply:
Thanks! Now done as you suggested in 529735b431b989d2acad2f46bd5983df7e72a360.